modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -35,20 +35,20 @@ Next you can https://community.exoscale.com/documentation/compute/custom-templat
 # Set the version and calcuate the checksum
 FCOS_VERSION='...'
 FILE="fedora-coreos-${FCOS_VERSION}-exoscale.x86_64.qcow2"
-CHECKSUM=$(md5sum $FILE | cut -d " " -f 1)
+CHECKSUM="$(md5sum "${FILE}" | cut -d " " -f 1)"
```

Comment:
I know `$()` allows you to nest things, but reading this still breaks my brain. I think we should just drop the outside quotes here:

```suggestion
CHECKSUM=$(md5sum "${FILE}" | cut -d " " -f 1)
```

Reply:
Should be fixed now.